food/parallel.js

```diff
@@ -143,11 +135,6 @@ d3.csv("nutrients.csv", function(raw_data) {
             var extent = yscale[d].brush.extent();
           }
 
-          // remove axis if dragged all the way left
-          if (dragging[d] < 12 || dragging[d] > w-12) {
-            remove_axis(d,g);
```

Comment:
ahh looks like it. can we make this into a `hide_axis` instead? or does D3 not have a sense of invisible objects?

Reply:
not that i know of. we would need to keep track of these object separately from D3 for reference later. When there is time, a thorough refactor is needed.